packages/react/.storybook/config.js

```diff
@@ -9,4 +10,47 @@ function loadStories() {
   req.keys().forEach(filename => req(filename));
 }
 
+addDecorator(centered);
+
+addParameters({
+  options: {
+    name: "Heetch Flamingo",
+    addonPanelInRight: true,
+    hierarchySeparator: /\//,
+  },
+});
+
+const flamingoViewports = {
+  BreakpointS: {
```

Comment:
Is this documented somewhere?
Shouldn't we use `mobile/tablet/desktop`?

Reply:
https://www.notion.so/heetch/Grid-1e18d12216b040c28d0e146125e7fded#dab7afa3ba2947f6a45a98e59d89d56a

Reply:
👍